ghc-lib-gen.cabal

```diff
@@ -12,16 +13,36 @@ copyright:           Digital Asset 2018-2019
 category:            Development
 build-type:          Simple
 extra-source-files:  ChangeLog.md
-cabal-version:       >=1.10
+
+library
+  build-depends:       base >=4.10
+                     , process >=1.6
+                     , filepath >=1.4
+                     , containers
+                     , directory
+                     , optparse-applicative
+                     , extra >=1.6
+  hs-source-dirs:    ghc-lib-gen/src
+  exposed-modules:   Ghclibgen
+                     GhclibgenOpts
+                     Paths_ghc_lib_gen
```

Comment:
I wouldn’t expose this, you aren’t actually using it outside of the lib.

Reply:
You are seeing the detritus of my desperation trying to get Cabal happy and those failing tests restored :) Fixed.